config.yaml

```diff
@@ -84,6 +84,12 @@ options:
     default:
     description: |
       The HTTPS proxy the registry server should use to access the upstream registry.
+  storage-cache:
+    type: string
+    default: "inmemory"
+    description: |
+      Cache provider for image layer metadata. Valid options are "inmemory" or "redis".
```

Comment:
Should we include an option such as `disabled` and block if the used value differs from the valid options (i.e., `inmemory` or `disabled`)? AFAIK, `redis` requires a working instance, which should be configured in the `redis` [structure](https://distribution.github.io/distribution/about/configuration/#redis). I fear that someone might set the value to `redis` and encounter a non-working registry.

Reply:
good catch! i'm now validating that we get expected values, or the charm will block.